ml/requirements.txt

```diff
@@ -0,0 +1,9 @@
+graphviz==0.10.1
+matplotlib==3.0.3
+numpy==1.16.2
+psycopg2==2.7.7
+psycopg2-binary==2.7.7
+pygraphviz==1.5
+sklearn==0.0
+xgboost==0.82
+pandas==0.24.1
```

Comment:
These are the packages required to run the XGBoost script. This is the usual convention for listing out the files used by the python script. With this file, one can easily set up the environments with `pip3 install -r requirements.txt`, and this is the convention. I could make it so that it is used in the xgboost shell script. Thanks for pointing out!

Reply:
Yes, then it would be good to insert comments in this file, if possible.